core/contracts/oracle/implementation/VoteTiming.sol

```diff
@@ -4,7 +4,7 @@ import "@openzeppelin/contracts/math/SafeMath.sol";
 import "../interfaces/VotingInterface.sol";
 
 /**
- * @title Library to compute rounds and phases for an equal length commit-reveal voting cycle. 
+ * @title Library to compute rounds and phases for an equal length commit-reveal voting cycle.
```

Comment:
Can't add a comment below to the `computeCurrentRoundId` method because you didn't change it, but missing a @return comment